src/program/lwaftr/run_nohw/run_nohw.lua

```diff
@@ -31,63 +37,81 @@ local function parse_args(args)
          check(file_exists(arg), "no such file '%s'", arg)
          conf_file = arg
       end;
+      b = function (arg)
+         check(arg, "argument to '--bt' not specified")
+         check(file_exists(arg), "no such file '%s'", arg)
+         bt_file = arg
+      end;
       B = function (arg)
          check(arg, "argument to '--b4-if' not specified")
-         b4_if = arg
+         b4_if_kind, b4_if = arg:match("^([a-z]+):([^%s]+)$")
+         check(b4_if,
+               "invalid/missing device name in '%s'", arg)
+         check(b4_if_kind and device_kind_map[b4_if_kind],
+               "invalid/missing device kind in '%s'", arg)
+         b4_if_kind = device_kind_map[b4_if_kind]
       end;
       I = function (arg)
          check(arg, "argument to '--inet-if' not specified")
-         inet_if = arg
+         inet_if_kind, inet_if = arg:match("^([a-z]+):([^%s]+)$")
+         check(inet_if,
+               "invalid/missing device name in '%s'", arg)
+         check(inet_if_kind and device_kind_map[inet_if_kind],
+               "invalid/missing device kind in '%s'", arg)
+         inet_if_kind = device_kind_map[inet_if_kind]
+      end;
+      D = function ()
```

Comment:
'D' flag is used for duration in many of the lwaftr commands and other snabb commands. Is it possible to use other name? In addition to it, I see the what this flag does is to set `conf.debug`, so the same can be achieved through a lwAFTR.conf file that has `debug` set to true. If that's the case, I'd prefer to remove this flag as a parameter option.